packages/fast-components/src/slider-label/slider-label.styles.ts

```diff
@@ -0,0 +1,58 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+
+export const SliderLabelStyles = css`
+    ${display("block")} :host {
+        --half-thumb-size: calc(${heightNumber} * 0.25);
+    }
+    .slider-label {
```

Comment:
needed?

Reply:
removed, it was used but only once, if we are talking about --half-thumb-size